docker_xylem/service.py

```diff
@@ -40,6 +45,27 @@ def xylem_request(self, queue, call, data):
             data=json.dumps(data),
         )
 
+    def _get_path(self, name):
+        return os.path.join(self.volume_path, name)
+
+    def _get_proc_mounts(self):
+        return open('/proc/mounts', 'rt').read()
+
+    def _check_mount(self, name):
+        path = os.path.join(self.mount_path, name)
+
+        mounts = self._get_proc_mounts()
+
+        mnt = None
+
+        for l in mounts.split('\n'):
+            if l.strip():
+                src, mount, typ, opts, dump, fpass = l.strip().split()
+                if mount == path:
+                    mnt = src
+
+        return mnt
```

Comment:
Also, this appears to only be used in a boolean context. Should it `return True` and `return False` instead?